utils/repository.py

```diff
@@ -0,0 +1,13 @@
+def convert_list_to_tuple_string(ids_list):
+    validate_list(ids_list)
+    id_value = (
```

Comment:
You could use `result` as a temp variable to be returned.

Reply:
result instead of id_value?